### ISSUE TITLE
[Bug]: Celo composer create command fails

### ISSUE THREAD

**Issue by Emmanuel-umeh** [celo-org/celo-composer#168]
### Which Framework?

React (with Tailwind)

### What happened?

The command `npx @celo/celo-composer create` fails with the following error when i select the frontend framework 'React with Tailwindcss (blank boilerplate)'.
```
Updating files: 100% (7/7), done.
cat: no such file or directory: packages/react-app-tailwindcss/package.json
Unhandled exception. Shutting down SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Command.createAsync (/Users/emmanuel/.npm/_npx/500f2d7f75beab14/node_modules/@celo/celo-composer/bin/create.js:191:35)

```
![Screenshot 2023-01-11 at 10 58 43](https://user-images.githubusercontent.com/40989414/211776039-22c6c09d-8066-4f80-9e4c-65090368da63.png)

#### Machine : Apple silicon m1

Also recreated this issue on a windows machine.

### Steps to reproduce

- run the command ```npx @celo/celo-composer@latest create```
- Choose front-end framework: React with Tailwindcss (blank boilerplate)
- Choose smart-contract framework: Hardhat
- Create a subgraph: No
- Project name : _insert your project name_

### Anything else?

Link to composer: https://github.com/celo-org/celo-composer

**Comment by Emmanuel-umeh** [comment]
could anyone recreate or resolve this issue?

**Comment by viral-sangani** [comment]
@Emmanuel-umeh This issue is fixed in v1.0.2.
Can you check with latest version and let me know if you still face this issue?

**Comment by Emmanuel-umeh** [comment]
yes it works fine in the new build. Thanks a lot💪🏼
